source/playbooks/SC/ssmdocs/scripts/sc_get_input_values.py

```diff
@@ -0,0 +1,136 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+SC_mappings = {
```

Comment:
I don't think I've ever even seen this bit of code. Thanks for fixing this.

I think you'll want to check the specific mapping though, e.g. CloudWatch.1 is for root user. This page has the security control IDs correlated to the other standards: https://docs.aws.amazon.com/securityhub/latest/userguide/cloudwatch-controls.html

Reply:
Good call out, it was setup to be 1 to 1 so I didn't realize that.